CHANGELOG.rst

```diff
@@ -85,6 +85,9 @@ Bugfixes
 - ec2_vpc_net - fix a bug where the module would get stuck if DNS options were updated in check mode (https://github.com/ansible/ansible/issues/62677).
 - elb_classic_lb - modify the return value of _format_listeners method to resolve a failure creating https listeners (https://github.com/ansible-collections/amazon.aws/pull/860).
 
+v3.3.1
+======
+
```

Comment:
We should probably add a release note here.  Even if 3.3.1 won't explicitly see it, it'll be available for later versions.  (needs to go into changelog.yaml and then re-run docs generation)